app/models/profile_field.rb

```diff
@@ -1,5 +1,5 @@
 class ProfileField < ApplicationRecord
-  include ActsAsProfileField
+  WORD_REGEX = /\b\w+\b/.freeze
```

Comment:
All the changes in this file are related to inlining the old `ActsAsProfileField` concern that was used to share functionality between `ProfileField` and `CustomProfileField`.